projects/output-and-error-handling/client/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+    "bufio"
```

Comment:
I'd recommend setting up your editor to auto-run `gofmt` to format the code on save, so that it's always consistent :)